### ISSUE TITLE
the charging order is right?

### ISSUE THREAD

**Issue by lalala215** [wsyCUHK/Reinforcement-Learning-for-Real-time-Pricing-and-Scheduling-Control-in-EV-Charging-Stations#6]
https://github.com/wsyCUHK/Reinforcement-Learning-for-Real-time-Pricing-and-Scheduling-Control-in-EV-Charging-Stations/blob/cbd4c8b3be01f779bb2c7e9fad9cc2c822b49442/code/HSA.py#L77-L83

Dear Author
I am wondering If this order is right?
` order=[operator.itemgetter(0)(t)-1 for t in sorted(enumerate(least,1), key=operator.itemgetter(1), reverse=True)] `
 
residual_demand[:,1] is parking time
residual_demand[:,0] is charging demand

_order_ put the biggest in _least_  as  the most need to be charged.
But the biggest in _least_ means its parking time is much larger than charging demand, I thought in this case  the biggest in _least_ means it can wait more time, so cannot put it as the most urgent.

maybe the reserse should be False, such as:
` order=[operator.itemgetter(0)(t)-1 for t in sorted(enumerate(least,1), key=operator.itemgetter(1), reverse=False)] `

**Comment by wsyCUHK** [comment]
Yes,  the reverse should be False. Thank you for pointing out the error. I will check the code recently.

**Comment by lalala215** [comment]
OK！
And I also have other two questions:
1. this line 38 no need for indentation
https://github.com/wsyCUHK/Reinforcement-Learning-for-Real-time-Pricing-and-Scheduling-Control-in-EV-Charging-Stations/blob/cbd4c8b3be01f779bb2c7e9fad9cc2c822b49442/code/HSA.py#L36-L43

2. In the paper feature2(Eq.(15)) is electricity price multiply total charging rate
f_{2}\left(S_{t}, A_{t}\right)=-c_{t} e_{t}
However in line 115 the feature2 is action[1], i.e. the total charging rate.
https://github.com/wsyCUHK/Reinforcement-Learning-for-Real-time-Pricing-and-Scheduling-Control-in-EV-Charging-Stations/blob/cbd4c8b3be01f779bb2c7e9fad9cc2c822b49442/code/HSA.py#L115
